### ISSUE TITLE
[Accessibility] improve output of screenreader after copying link to clipboard

### ISSUE THREAD

**Issue by hassmal** [Dataport/terminfinder-frontend#331]
# Story

After creating a survey there is an option to copy the link to clipboard. If one does that while using a screenreader, the output is not helpful. Additionally the focus is reset to the start of the page and not visible anymore.

Here is the output of Apple VoiceOver:
 
![Bildschirmfoto 2024-10-02 um 08 40 50](https://github.com/user-attachments/assets/a95ffe8d-723d-401d-9ef3-d7084e48529e)
